flight/mcu/lib/ServoControl/ServoControl.cpp

```diff
@@ -32,25 +35,22 @@ void ServoControl::roundOffAngle(float& value) {
  * @param length_two The length of the flap extension for the clockwise flaps.
  *
  */
-void ServoControl::servoActuation(float length_one, float length_two) {
-    // These are correcting factors for finding the angle. We still need to
-    // calculate what these values are. These are placeholders for now. m and
-    // offset should include the radian to degree conversion.
-    float m = (1.0 / radius) * (180.0 / (3.1415));  // degrees / meter
-    float offset = 0;
+void ServoControl::servoActuation(float length) {
+    // The angle is found through utilizing a fft and mapping extension/angle
+    // values to a sine function. len (mm), pass in ang (rad)
 
-    // This is the actual conversion from the inputted length to the angles
-    // desired for the servos.
-    float ccw_angle = (length_one * m + offset);
-    float cw_angle = (length_two * m + offset);
+    if (length < 0) length = 0;
+    if (length > 0.018) length = 0.018;
 
-    roundOffAngle(cw_angle);
-    roundOffAngle(ccw_angle);
+    /* Maps the length to an angle based on calibration */
+    float angle = -0.035 + 1.09 * pow(10, 3) * length +
```

Comment:
Since their coefficients for a polyfit, they're very very specific to the current flap hardware, the type of rubber band etc. They probably will not have to be changed unless we switch hardware, in which case the degree of the polyfit etc might change anyways. I will leave it like this for now, but we should look into better ways of representing polyfits since **some people** are very prone to using them.